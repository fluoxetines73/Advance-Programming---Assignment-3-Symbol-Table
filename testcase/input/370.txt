BEGIN
ASSIGN g_1 unimportant
END random
INSERT a string
ASSIGN a 'abcABC'
UWAAAA
END
END
PRINT