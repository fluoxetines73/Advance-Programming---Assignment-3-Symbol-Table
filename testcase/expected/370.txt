Undeclared: ASSIGN g_1 unimportant